builtin/providers/softlayer/resource_softlayer_network_loadbalancer_virtualipaddress_test.go

```diff
@@ -21,22 +18,16 @@ func TestAccSoftLayerVirtualIpAddress_Basic(t *testing.T) {
 			resource.TestStep{
 				Config: testAccCheckSoftLayerVirtualIpAddressConfig_basic,
 				Check: resource.ComposeTestCheckFunc(
-					testAccCheckSoftLayerVirtualIpAddressExists("softlayer_network_loadbalancer_virtualipaddress.testacc_vip", &vip),
```

Comment:
Using the `resource.TestCheckResourceAttr(
						"softlayer_network_loadbalancer_virtualipaddress.testacc_vip", "name", "test_load_balancer_vip"),` line